acpi/src/madt.rs

```diff
@@ -551,12 +551,15 @@ pub struct GiccEntry {
     performance_interrupt_gsiv: u32,
     parked_address: u64,
     gic_registers_address: u64,
-    gic_control_block_address: u64,
+    gic_virtual_registers_address: u64,
+    gic_hypervisor_registers_address: u64,
```

Comment:
Interesting, do you know if this field was retrospectively added (I'm guessing not) or if we just missed it when we first wrote this (more likely)?

Reply:
I suspect it was just skipped accidentally when the struct was created. I stumbled across it because I noticed every `GiccEntry` returned appeared to have an 'MPIDR' field of `0`. :slightly_smiling_face: